config/project_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "covid19-som": {
+    "metadataPath": "datasets/covid19-som/metadata/chart-config",
+    "allowedEmailDomains": "lark.systems|africasvoices.org"
```

Comment:
Not sure if I've mentioned this before, but you could consider using a list instead of a regex string - if I remember correctly, in the dart code you separate it into a list anyway, so that would just make it easier.

Reply:
Yes this is much easier to read, fixed.